src/store/actions/transactions.js

```diff
@@ -68,12 +68,16 @@ export const transactionsRetrieved = ({
           count: response.meta.total,
         },
       });
+      console.log('Success')
```

Comment:
Please remove the console logs from all the places you have added 🙏🏼

Reply:
Done